src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -176,12 +176,12 @@ export class CancerSummaryChart extends React.Component<CancerSummaryChartProps,
                         {
                             target: "data",
                             mutation: (props:any) => {
-                                if (props.datum.x in self.props.countsByGroup) {
+                                if (props.datum.xKey in self.props.countsByGroup) {
```

Comment:
When viewing data by cancer study, we need to group by by studyId instead of by label.  This was caused by a change (probably improvement) to logic in minor version of Victory.